types/items/DatepickerFieldModel.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021. Sendanor <info@sendanor.fi>. All rights reserved.
+
+import { FormItemType } from "../FormItemType";
+import { FormFieldModel,  isFormFieldModel } from "../FormFieldModel";
+
+export interface DatePickerFieldModel extends FormFieldModel {
+
+    readonly type         : FormItemType.DATE_FIELD;
+    readonly minLength   ?: number;
+    readonly maxLength   ?: number;
+
+}
+
+export function isDatepickerFieldModel (value: any) : value is DatePickerFieldModel {
+    return value?.type === FormItemType.DATE_FIELD && isFormFieldModel(value);
```

Comment:
This should check for `minLength` and `maxLength`, although not sure why DatePicker would have those at all.

Reply:
removed minLength and maxLength values